lighthouse-core/test/audits/unsized-images-test.js

```diff
@@ -219,8 +255,10 @@ describe('Sized images audit', () => {
       const result = await runAudit({
         attributeWidth: '100',
         attributeHeight: '100',
-        cssWidth: '',
-        cssHeight: '100',
+        cssSizing: {
+          width: null,
```

Comment:
if width is not set.... didn't we want `width: undefined`? The width wasn't set anywhere, it's not that it was explicitly set to null/nothing/whatever.

Reply:
heh, well, what you're saying now is consistent with my interpretation of  your earlier comment. :)

however.. i don't agree with those semantics and agree with the positioning that patrick offered:

> I tend to think of `undefined` as equivalent to "not set" and `null` as equivalent to "set and has no value".

specifically in this case... `.cssSizing.width === null` means that we did gather cssSizing data and looked through all matched rules and there's no width in those rules.

Reply:
> specifically in this case... .cssSizing.width === null means that we did gather cssSizing data and looked through all matched rules and there's no width in those rules.

In other words, it wasn't set? :p 

> however.. i don't agree with those semantics and agree with the positioning that patrick offered:

I'm operating under the same guideline. It's interesting we seem to interpret it differently here.

Let me trying changing a word ...

> I tend to think of undefined as equivalent to "not declared" and null as equivalent to "declared and has no value".

A developer creates an image element. It has no explicit width set in HTML nor CSS. They didn't declare it. So the value is undefined. In this case, "declare that it has no value" has no meaning (that's just the way this property is).

___

After thinking more on it, since this isn't really an explicit property (cssSizing.width/height does not exist anywhere on the DOM), and is really an emergent property (do a bunch of logic to summarize "size as defined by style"), the whole idea of "not set" becomes meaningless, so I cede to `null` :)